settings.gradle

```diff
@@ -6,7 +6,7 @@ pluginManagement {
 }
 
 plugins {
-	id "com.gradle.enterprise" version "3.5.1"
+	id "com.gradle.enterprise" version "3.7.2"
```

Comment:
@jprinet Is there a specific reason to bump to `3.7.2`? Typically, `io.spring.ge.conventions` is bumped when `com.gradle.enterprise` is bumped.

Reply:
Hi @jgrandja and sorry for the late response.

I don't know how your build logic is implemented but I know for sure that your plugin version is outdated:
https://ge.spring.io/s/d5rbpayhwdho4/plugins#com.gradle.enterprise-22

Here you can find the accurate list of improvements and fixes since 3.5.1:
https://docs.gradle.com/enterprise/gradle-plugin/#release_history